src/submit.py

```diff
@@ -83,6 +83,22 @@ async def _check_git_repo(repo_dir: pathlib.Path):
 
     await _create_submission_commit(repo_dir)
 
+    if "full internet" not in settings.get_settings().get("task", {}).get(
+        "permissions", []
+    ):
+        click.confirm(
+            "\n"
+            "Since this task is running on a container with no internet access, "
```

Comment:
Don't need to mention containers. "This task environment does not have internet access"